.github/workflows/main.yml

```diff
@@ -29,60 +31,58 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-14]
+        os: [ubuntu-latest, windows-latest, macos-latest]
     steps:
-      - name: Set up repository
-        uses: tree-sitter/parser-setup-action@v1.2
-        with:
-          node-version: 20
+      - name: Clone repository
+        uses: actions/checkout@v4
       - name: Clone nvim help files
         uses: actions/checkout@v4
         with:
           repository: neovim/neovim
           path: examples/neovim
           sparse-checkout: runtime/doc/
-      - name: Run tests
+      - name: Set up tree-sitter
+        uses: tree-sitter/setup-action/cli@v1
+      - name: Run parser tests
         uses: tree-sitter/parser-test-action@v2
         with:
-          test-library: ${{runner.os == 'Linux'}}
-          corpus-files: |-
+          test-rust: ${{runner.os == 'Linux'}}
```

Comment:
I don't get this. Why only linux? Is it to save CI time?

Reply:
Yep. I don't mind setting it to either `false` or `true` though.